apps/filebrowser/src/filebrowser/views.py

```diff
@@ -250,6 +250,11 @@ def view(request, path):
           '/filebrowser/view=' + urllib_quote(trash_path.encode('utf-8'), safe=SAFE_CHARACTERS_URI_COMPONENTS)
       )
 
+  if sys.version_info[0] > 2:
+    _is_ajax = request.headers.get('x-requested-with') == 'XMLHttpRequest'
+  else:
+    _is_ajax = request.is_ajax()
+
```

Comment:
Let's not pollute the code too much this way, how about adding a `is_ajax(request)` function in `desktop/core/src/desktop/lib/view_util.py` and just call it later?

Reply:
done @romainr